Formula/postgresql.rb

```diff
@@ -94,11 +94,13 @@ def install
   end
 
   def post_install
+    require "open3"
     (var/"log").mkpath
     (var/"postgres").mkpath
     unless File.exist? "#{var}/postgres/PG_VERSION"
       system "#{bin}/initdb", "#{var}/postgres"
     end
+    _stdout, _stderr, _status = Open3.capture3("createdb")
```

Comment:
@MikeMcQuaid the command may lead to displaying the error: 
`createdb: database creation failed: ERROR:  database "harshn" already exists`
I think displaying the error is not required, and the command doesn't change anything if the db exists already. (in this case harshn) That's why I thought opting for Open3 instead of system would be better.